src/main/java/TicTacToe/Player.java

```diff
@@ -18,4 +18,14 @@ public String getMark() {
     public String getInput() {
         return display.getInput();
     }
+
+    @Override
+    public String getMarkChoice() {
```

Comment:
Is it important to have separate `getInput()` and `getMarkChoice()` methods? In both instances we're asking the player to provide input, right?